starter-code/MyLinkedList.java

```diff
@@ -13,50 +13,118 @@ public int getSize() {
 
 	public boolean remove(int index) {
 		//to-do
+
+		/*
+		head
+		 */
+		if (index == 0) {
+
+			head = head.getNext();
```

Comment:
Yes! This case is easy to overlook, so I'm glad you considered it! Just remember to check if `head` itself is null (which it would be if size is zero) before calling `getNext()` on it.